### PR TITLE
Hours fix

### PR DESCRIPTION
**Pull request by ollieeknight**
_Should_ allow users to submit jobs with hours not days

### CODE REVIEW

File: form.yml [hunk 1]
```diff
@@ -51,21 +50,19 @@ attributes:
...
-    widget: "number_field"
-    label: "Running time [days]"
-    value: 1
-    help: Number of days to run.
-    min: 1
-    max: 14
-    step: 1
+  bih_time:
+    widget: "text_field"
+    label: "Running time"
```
Comment (sellth) [inline form.yml:55]: Please set a default value here and make it a required field. 
```
value: "1d"
required: true
```

File: submit.yml.erb [hunk 2]
```diff
@@ -1,7 +1,12 @@
-<%-
+<%- 
   cpus = bih_cpus.to_i
   mem = bih_mem.to_i
-  days = bih_days.to_i
+  time = bih_time
+  if time.include?("h")
```
Comment (sellth) [inline submit.yml.erb]: time.end_with("h") seems more appropriate here.

File: submit.yml.erb [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
+    formatted_time = "0-" + time.gsub("h", "")
```
Comment (sellth) [inline submit.yml.erb]: time.delete_suffix("h") seems like a more defined operation.

File: submit.yml.erb [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
+  elsif time.include?("d")
+    formatted_time = time.gsub("d", "") + "-0"
```
Comment (sellth) [inline submit.yml.erb]: same as above

**Review by sellth — CHANGES_REQUESTED** [review]
Nice, this should work. Please make a few changes and I'll merge.

**Comment by ollieeknight** [comment]
Thomas, is this now correct? I've never responded to a pull request review before, but I made the changes and pushed it to the branch.

File: submit.yml.erb [hunk 5]
```diff
@@ -2,10 +2,10 @@
   cpus = bih_cpus.to_i
   mem = bih_mem.to_i
   time = bih_time
-  if time.include?("h")
-    formatted_time = "0-" + time.gsub("h", "")
+  if time.end_with?("h")
+    formatted_time = "0-" + time.delete_suffix("h")
   elsif time.include?("d")
```
Comment (sellth) [inline submit.yml.erb]: Hi Olli, thanks for following up. This should also be "time.end_with".

Comment (ollieeknight) [reply submit.yml.erb]: great - should be updated

**Pull request merged**